text/0000-north-star.md

```diff
@@ -0,0 +1,466 @@
+- Feature Name: north_star
+- Start Date: 2016-08-07
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+A refinement of the Rust planning and reporting process, to establish a shared
+vision of the project among contributors, to make clear the roadmap toward that
+vision, and to celebrate our achievements.
+
+Rust's roadmap will be established in year-long cycles, where we identify up
+front - together, as a project - the most critical problems facing the language
+and its ecosystem, along with the story we want to be able to tell the world
+about Rust. Work toward solving those problems, our short-term goals, will be
+decided in quarter-long cycles by individual teams. For the purposes of
+reporting the project roadmap, goals will be assigned to quartely milestones,
```

Comment:
At a project-level, I'm not sure how useful it is to set quarterly milestones - these things are hard to estimate and we only risk bad publicity and disappointment for not meeting them. I'd rather see relative time estimates of goals, i.e., "you can expect to see feature a delivered before feature b, and feature a is a high priority"